crypto/Pos/Arbitrary/Crypto.hs

```diff
@@ -224,3 +230,86 @@ instance Arbitrary HDPassphrase where
 
 instance Arbitrary HDAddressPayload where
     arbitrary = genericArbitrary
+
+----------------------------------------------------------------------------
+-- WithHash
+----------------------------------------------------------------------------
+
+instance (Bi a, Arbitrary a) => Arbitrary (WithHash a) where
+    arbitrary = do
+        a <- arbitrary
+        return $ withHash a
+    shrink = genericShrink
+
+----------------------------------------------------------------------------
+-- Ed25519
+----------------------------------------------------------------------------
+
+instance Arbitrary Ed25519.PointCompressed where
+    arbitrary = do
+        a <- arbitrary
+        return $ Ed25519.pointCompressed a
+
+instance Arbitrary Ed25519.Scalar where
+    arbitrary = do
+        a <- arbitrary
+        return $ Ed25519.scalar a
+
+instance Arbitrary Ed25519.Signature where
+    arbitrary = do
+        a <- arbitrary
+        return $ Ed25519.Signature a
+
+---------------------------------------------------------------------------
+-- Cardano.Crypto.Wallet
+---------------------------------------------------------------------------
+
+instance Arbitrary CC.ChainCode where
+    arbitrary = do
+        a <- arbitrary
+        return $ CC.ChainCode a
+
+instance Arbitrary CC.XPrv where
+    arbitrary = do
+        a <- arbitrary
+        return $ CC.xprv b
+
```

Comment:
@gromakovsky how do I define arbitrary instance for XPrv which is 
```
newtype XPrv = XPrv EncryptedKey
    deriving (NFData, ByteArrayAccess)
```
The constructor is not exported and it is defined as external git dependency so I should not touch the module.
I can work with `xprv` exported function which returns `Either String XPrv` but I don't know how would the `Left` case look then ?